src/oscar/apps/payment/bankcards.py

```diff
@@ -32,6 +33,7 @@
     (VISA, (13, 16), ('4',)),
     (VISA_ELECTRON, (16,), ('4026', '417500', '4405', '4508',
                             '4844', '4913', '4917')),
+    (MIR, (16), ('2')),
```

Comment:
Both these need to be tuples, for which a trailing comma is required inside each bracket. 

Also from [the Wikipedia entry](https://en.wikipedia.org/wiki/Payment_card_number) for this, the prefixes are a bit more complex than just `2` - I think an approach similar to what has been used for Discover above is required.